internal/pubsub/example_test.go

```diff
@@ -78,8 +79,11 @@ func Example_sendReceiveMultipleMessages() {
 
 	// Receive messages from the subscription.
 	ms2 := []string{}
-	for i := 0; i < len(ms); i++ {
+	for {
 		m2, err := s.Receive(ctx)
+		if err == context.DeadlineExceeded {
+			break
+		}
```

Comment:
I can't comment on line 91 because it's not part of the diff, but to be more like a real example, the ack should happen after you process the message, which in this case is after you print it.

Reply:
Sorry, my mistake. You're right, it's fine as is.